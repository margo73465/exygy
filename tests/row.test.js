import React from 'react';
import Row from '../components/row.jsx';
import renderer from 'react-test-renderer';

test('renders a row', () => {
  const rowData = {
    city: "Alameda",
    county: "Alameda County",
    median_list_rent: "2087",
    median_list_rent_ia: "2087",
    median_list_rent_ia_percentchg_1994: "0.4808",
    median_list_rent_percentchg_1994: "1.575",
    source: "real Answers",
    year: "2015"
  };
  const component = renderer.create(<Row row={rowData}/>)
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})