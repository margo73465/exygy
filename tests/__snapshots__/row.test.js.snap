// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a row 1`] = `
<tr
  className="row"
>
  <td
    data-label="City"
  >
    Alameda
  </td>
  <td
    data-label="County"
  >
    Alameda County
  </td>
  <td
    data-label="Year"
  >
    2015
  </td>
  <td
    data-label="Median List Rent"
  >
    2087
  </td>
  <td
    data-label="Median List Rent (IA)"
  >
    2087
  </td>
  <td
    data-label="Median List Rent (% Change Since 1994)"
  >
    1.575
  </td>
  <td
    data-label="Median List Rent (IA % Change Since 1994)"
  >
    0.4808
  </td>
</tr>
`;
